src/main/java/Mariogatchi/RemoveItemRunner.java

```diff
@@ -0,0 +1,31 @@
+package Mariogatchi;
+
+public class RemoveItemRunner implements RemoveItemInputBoundary{
+
+    private final RemoveItemPresenter presenter;
+
+    /*
+    The interactor for the UseItem use case
+    @param presenter - the presenter for the RemoveItem use case
+     */
+    public RemoveItemRunner(RemoveItemPresenter presenter) {
+        this.presenter = presenter;
+    }
+
+    /*
+    Removes a specified amount of an item from an inventory
+    returns an instance of RemoveItemResponseModel
+     */
+    @Override
+    public RemoveItemResponseModel removeItemFromInv(RemoveItemRequestModel requestModel) {
+        if (requestModel.getInventory().removeItem(requestModel.getItem(), requestModel.getQuantity())) {
```

Comment:
Should you be able to remove Reusable items from your inventory. Could break some game functionality.